src/servers/public-api/fines/fines.controller.ts

```diff
@@ -63,6 +63,21 @@ export class FinesController {
     }
   }
 
+  @Get('/')
+  async list(
+    @Request() request: ExpressRequest,
+    @Response() response: ExpressResponse,
+    @Next() next: ExpressNextFunction
+  ) {
+    try {
+      const fine = await services.fines.list(request);
```

Comment:
Methods in service are intended to be reusable throughout the codebase / application.
Currently it's being tightly coupled with the Express Request context, please identify the key values that are required in listing. 

It should be `page`, `limit` or perhaps an optional `keywords` for searching. 

By doing so you could reuse the `list` method for example, for excel exporting